.github/workflows/ci.yml

```diff
@@ -14,11 +14,11 @@ jobs:
   test:
     runs-on: ubuntu-latest-16-cores
     container:
-      image: ghcr.io/dojoengine/dojo-dev:136a67a
+      image: ghcr.io/dojoengine/dojo-dev:bc6a5c1
     steps:
       - uses: actions/checkout@v3
       - uses: Swatinem/rust-cache@v2
-      - run: cargo llvm-cov nextest --all-features --lcov --output-path lcov.info
+      - run: cargo build -r --bin katana && cargo llvm-cov nextest --all-features --lcov --output-path lcov.info
```

Comment:
why is it necessary to build katana here, shouldn't it be available in the dev container?

Reply:
ah is this since you made the account number change to katana? ideally we merge that as a separate pr and recreate the dev container so we can avoid adding the build overhead to the test runner. or maybe it doesn't add too much time and it is worth always running against the latest?

Reply:
Separate PR is definitely preferable, this is a temporary solution